x-pack/solutions/security/packages/features/src/notes/kibana_features.ts

```diff
@@ -36,8 +36,8 @@ export const getNotesBaseKibanaFeature = (
         all: params.savedObjects,
         read: params.savedObjects,
       },
-      ui: ['read', 'crud'],
-      api: ['notes_read', 'notes_write'],
+      ui: [],
```

Comment:
These are used to check for the notes featurye availability and should not be removed. (search for `securitySolutionNotes.read` in Kibana)